include/simdjson/generic/ondemand/array.h

```diff
@@ -43,6 +43,10 @@ class array {
    * safe to continue.
    */
   simdjson_really_inline simdjson_result<size_t> count_elements() & noexcept;
+
+  inline simdjson_result<value> at_pointer(std::string_view json_pointer) noexcept;
```

Comment:
If this is a public method then it should be described with a comment above.

Reply:
(Note that the DOM equivalent was surely commented already, so some reuse is possible.)

Reply:
@NicolasJiaxin Please add a remark that goes something like this: "JSON Pointer key access on objected is resolved using the find_field method."